x-pack/heartbeat/Jenkinsfile.yml

```diff
@@ -27,6 +27,9 @@ stages:
     goIntegTest:
         mage: "mage goIntegTest"
         stage: mandatory
+    validateIronbank:
```

Comment:
This stage is the one to validate the ironbank for `heartbeat` are compliance. If it fails then, look at the README.md in the `ironbank` folder that contains enough details to handle what's required to be done